site.hs

```diff
@@ -20,24 +20,24 @@ main = hakyll $ do
         compile copyFileCompiler
 
     match "contributors.txt" $ do
-            route   idRoute
-            compile copyFileCompiler
+        route   idRoute
+        compile copyFileCompiler
```

Comment:
if this reformatting is intentional it would be better to have it in a seperate commit to have this PR as a coherent unit

Reply:
Done